README.md

```diff
@@ -1,5 +1,12 @@
 External user authentication
 ============================
+
+**⚠⚠ Warning:** As of Version 3.0 this app uses namespace \OCA\UserExternal now. You MUST change your config to adopt to this change. ⚠⚠
+
+## Usage
+
+Go to Apps-Folder of your Nextcloud Installation
```

Comment:
@MarBie77 what's the meaning of this phrase? 
the configuration has to be done in `config/config.php`...

Reply:
Sorry, removed the additional text from a former version!